examples/js/ada/get-pledge-vkeys.js

```diff
@@ -0,0 +1,61 @@
+const BitGoJS = require('bitgo');
+const accountLib = require('@bitgo/account-lib');
+const bitgo = new BitGoJS.BitGo({ env: 'test' });
+const Promise = require('bluebird');
+const coin = 'tada';
+const basecoin = bitgo.coin(coin);
+const fs = require('fs');
+
+// TODO: set your access token here
+const accessToken = '';
+// TODO: set your wallet ID here
+const walletId = '';
+
+Promise.coroutine(function *() {
+  bitgo.authenticateWithAccessToken({ accessToken: accessToken });
+  yield bitgo.unlock({ otp: '000000', duration: 3600 });
+  const walletInstance = yield basecoin.wallets().get({ id: walletId });
+
+  console.log('Wallet ID:', walletInstance.id());
+
+  if (!walletInstance._wallet.keys || walletInstance._wallet.keys.length !== 3) {
```

Comment:
This is awesome! Will all wallets have 3 keys associated? Or only newly created ones?

Reply:
All ADA wallets shall have 3 keys associated